pkg/zeaburpack/main.go

```diff
@@ -173,16 +175,36 @@ func Build(opt *BuildOptions) error {
 		return err
 	}
 
+	_ = os.RemoveAll(".zeabur")
+
+	if t == types.PlanTypeNodejs && m["framework"] == string(types.NodeProjectFrameworkNextJs) && m["serverless"] == "true" {
+		println("Transforming build output to serverless format ...")
+		err = nextjs.TransformServerless(*opt.ResultImage, *opt.Path)
+		if err != nil {
+			println("Failed to transform serverless: " + err.Error())
```

Comment:
```suggestion
			log.Println("Failed to transform serverless: " + err.Error())
```

println is not recommended by Go team :(

Reply:
https://github.com/zeabur/zbpack/pull/135/commits/98fdd4a967173ff576500018f7718474face7bdd